servers/rendering/renderer_rd/shaders/color_space_inc.glsl

```diff
@@ -0,0 +1,32 @@
+
+vec3 srgb_to_linear(vec3 color) {
+	// This expects 0-1 range input, outside that range it behaves poorly.
+	// Approximation from http://chilliant.blogspot.com/2012/08/srgb-approximations-for-hlsl.html
+	return color * (color * (color * 0.305306011 + 0.682171111) + 0.012522878);
```

Comment:
Why is `srgb_to_linear` changed to an approximation instead of the more accurate conversion that was there before?

Reply:
I picked this up from the GLES3 renderer, it seems to use this one exclusively while RD uses the other form.  Since HDR mode is likely going to only be an RD feature, I think I'll put it back.

Reply:
This change has been reverted to match what was in use by the RD renderers before.